app/lib/features/chat_ng/widgets/message_timestamp_widget.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:acter/common/themes/acter_theme.dart';
+import 'package:flutter/material.dart';
+import 'package:acter/features/chat_ng/utils.dart';
+
+class MessageTimestampWidget extends StatelessWidget {
```

Comment:
wait, there is a widget. why isn't this used?

Reply:
It's been used in messages type widgets...